modules/yandexBidAdapter.js

```diff
@@ -166,6 +166,11 @@ export const spec = {
         queryParams['ssp-cur'] = currency;
       }
 
+      const lang = document.documentElement.getAttribute('lang');
```

Comment:
Can you put this in the request object for all adapters to get? I'd hate to see a bunch of repeated implementations